management/http_recordings_test.go

```diff
@@ -192,3 +216,21 @@ func redactSensitiveDataInResourceServer(t *testing.T, i *cassette.Interaction)
 		i.Response.Body = string(rsBody)
 	}
 }
+
+func redactSensitiveDataInLogSession(t *testing.T, i *cassette.Interaction) {
```

Comment:
This was missing when log sessions support was added to the SDK sadly. The recording was added manually at that time.